source/administration/bulk-export.rst

```diff
@@ -0,0 +1,35 @@
+.. _bulk-export:
+
+=================
+Bulk Export Tool
+=================
+
+Data from one Mattermost instance into another can be exported in the a `JSONL
+<http://jsonlines.org>`_ file using the `bulk loading feature <https://docs.mattermost.com/deployment/bulk-loading.html>`_. This tool is useful if you have created a server for a proof of concept and have created another server for production use and want to retain the history from the proof of concept instance. 
+
+You can export the following basic data types:
+
+- Version
+- Teams
+- Channels (Public & Private)
+- Users
+- Users' Team memberships
+- Users' Channel memberships
+- Users' notification preferences
+- Posts (regular, non-reply posts)
+
+.. include:: bulk-export-data.rst
+
+Running the bulk export command
+===============================
+The export command runs in the `CLI <https://docs.mattermost.com/administration/command-line-tools.html>`_ and operates in the security context of the CLI.  It has permissions to access all information in the Mattermost database. 
```

Comment:
What is the significance of `operates in the security context of the CLI.`

Reply:
Removed.  Took some of that from another page.